src/FlyWithLua.cpp

```diff
@@ -5721,6 +5855,11 @@ void RegisterCoreCFunctionsToLua(lua_State* L)
     lua_register(L, "save_situation", LuaSaveSituation);
     lua_register(L, "load_aircraft", LuaLoadAircraft);
     lua_register(L, "reload_scenery", LuaReloadScenery);
+    lua_register(L, "XPLMGetMouseLocationGlobal", LuaXPLMGetMouseLocationGlobal);
```

Comment:
I sugges to use different names than the native names. Otherwise this will produce name clashes when legacy scripts use FFI. Maybe FWL as prefix instead of XPLM.

Reply:
Let me know what the agreement on naming is and I will change, FWL sounds good, I could also just drop the XPLM...note this means changing all the other XPLM functions in FlyWithLua.cpp too.